### ISSUE TITLE
unprocessed components while importing solution

### ISSUE THREAD

**Issue by lar-mar** [WaelHamze/dyn365-ce-vsts-tasks#165]
When I use the import solution task I always get informed about unprocessed components:
![image](https://user-images.githubusercontent.com/15412588/78331648-ca667080-7586-11ea-9928-97c8fe836971.png)
In the build task I selected to "Publish Workflows":
![image](https://user-images.githubusercontent.com/15412588/78331989-6e501c00-7587-11ea-854a-ef817b1e01b5.png)

When I check the import logfile it tells me that several components could not processed. There is no error in the log.
![image](https://user-images.githubusercontent.com/15412588/78331713-e9fd9900-7586-11ea-9a53-e8d23c9ec147.png)

When I check the components in the target environment they are obviously processed because the processes are activated and up to date.

Do you have a hint why the message appears and how to solve it?

**Comment by WaelHamze** [comment]
@lar-mar 

If you download the import log file, you will notice that there will be a number of unprocessed components. I've created the warning to warn the user incase they want to take any action and investigate.

I have noticed in some scenarios the workflows don't publish, we I have some custom c# code and runs after the import the publishes any unpublished workflows.

Normally this would happen if you import workflows that have incorrect references to records. But I see it happening too when workflow is fine.
